cmd/aspect/query/query.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright © 2021 Aspect Build Systems
+
+Not licensed for re-use
+*/
+
+package query
+
+import (
+	"aspect.build/cli/pkg/aspect/query"
+	"aspect.build/cli/pkg/bazel"
+	"aspect.build/cli/pkg/ioutils"
+	"github.com/spf13/cobra"
+)
+
+func NewDefaultQueryCmd() *cobra.Command {
+	return NewQueryCommand(ioutils.DefaultStreams, bazel.New())
+}
+
+func NewQueryCommand(streams ioutils.Streams, bzl bazel.Bazel) *cobra.Command {
+	q := query.New(streams, bzl, true)
+
+	// Load these from somewhere
```

Comment:
Ill make it more clear. Basically the presets are hardcoded right there in that presets list. I think the comment was meant to mean that "some of these should be loaded from the plugin config". Which there is an open issue for btw so it will come in a later PR